aggie-coding-club/Rev-Registration
# Backend - async only filter

## Description

Adds the `asynchronous` field to the `Section` model (and serializer), sets the field during scraping.

## How to test

Run scrape_courses, then run the server and look at like `COMM 203` (or any other course with a bunch of async sections) and double check the route is returning `asynchronous: true` for the async sections.

Since this adds a column to the Section table, you'll need to run `python manage.py migrate`

Note that `/scheduler/generate` won't work since the `asynchronous` field isn't being sent by the frontend - this will be fixed in #389, and I'll be sure to wait on merging this until that part is done.

## Related tasks

Closes #388
